examples/electron/src-gen/frontend/electron-main.js

```diff
@@ -16,6 +16,7 @@ process.env.LC_NUMERIC = 'C';
 
 const electron = require('electron');
 const path = require('path');
+const backend = require("../backend/main")
```

Comment:
electron doc says that some APIs only available after `electron.app.on('ready'` if such APIs used by some backends modules it could break